### PR TITLE
Add support for responsive maps thru a 'responsive' property and a us…

### PR DESCRIPTION
**Pull request by eitanp461**
…ag example

### CODE REVIEW

File: src/datamap.jsx [hunk 1]
```diff
@@ -66,46 +78,50 @@ export default class Datamap extends React.Component {
...
-		if (!map) {
-			map = this.map = new Datamaps({
-				...props,
-				data,
-				element: this.refs.container
-			});
-		} else {
-			map.updateChoropleth(data, updateChoroplethOptions);
-		}
+	} = this.props;
```
Comment (btmills) [inline src/datamap.jsx]: It looks like the whitespace from here down got messed up. Can you fix that?

**Review by btmills — CHANGES_REQUESTED** [review]
This is great, thank you! Just one bit of cleanup, and then I think this is ready to go.

File: src/datamap.jsx (hunk 1 above)
Comment (eitanp461) [reply src/datamap.jsx]: No problem. VSCode has some issues with spread operator

**Review by btmills — APPROVED** [review]
LGTM, thanks! I'll get a release out shortly :ship:

**Comment by btmills** [comment]
Released as [v0.3.0](https://github.com/btmills/react-datamaps/releases/tag/v0.3.0) :shipit:

**Comment by eitanp461** [comment]
Thanks for merging this change and creating this module :)

**Pull request merged**